magic6/magic6.cpp

```diff
@@ -0,0 +1,83 @@
+#include <fstream>
+
+using namespace std;
+
+ifstream input("magic6.in");
+ofstream output("magic6.out");
+
+char character;
+int n;
+int frequencyVector[10] = {};
+
+void verifyIfItsIntegerOrChar()
+{
+	if (character > 47 && character < 58)
+	{
+		frequencyVector[character - '0'] = 1;
+	}
+	else
+	{
+		character = (char)(character - 32);
+		output << character;
```

Comment:
variable character is overwritten here
you should output directly (char)(character - 32)